docs/installation/configuration.rst

```diff
@@ -0,0 +1,63 @@
+.. _configuration:
+
+=======================
+Open Zaak configuration
+=======================
+
+Before you can work with Open Zaak after installation, a few settings need to be
+configured first.
+
+.. _configuration_sites:
+
+Setting Sites.Site
+==================
+
+In the admin, under **Configuration > Sites**, make sure to change the existing
+``Site`` to the domain under which Open Zaak will be deployed (see :ref:`manual_configuration` for more
+information).
+
+Configure API-access between components
+=======================================
+
+To allow communication between the different APIs in Open Zaak, e.g. between Zaken and Catalogi, etc.,
+follow these steps:
+
+- Create ``Client credentials`` for each component (**API Authorizations > Client credentials**)
+- For each component, add ``external API credentials`` for the other components that
+  it needs to connect to (**API Authorizations > External API credentials**)
+- Create ``Applications`` with the correct ``Authorizations`` for the ``client credentials``
+  created in the first step (see also :ref:`manual_api_auth`)
+
+For more detailed information on the steps above, see :ref:`manual_authorizations`.
+
+Configure Notifications API
+===========================
+
+Next, the Notifications for Open Zaak must be configured. Set the correct URL
+for the Notifications API in the admin and define the necessary Webhook subscriptions
+(see :ref:`manual_configuration`).
+
+
+Create an API token
+===================
+
+Create another ``Client credential`` and also create an ``Application`` for it. Add an ``Authorization``
+with the scope ``catalogi.lezen`` to this ``Application``. These
+credentials will be used to make the first request to the Open Zaak APIs. After creating the
+``Application``, copy the JSON Web Token (JWT) shown under **Client credentials**.
```

Comment:
Add a warning that this JWT has limited validity and should not be used in real applications.